sdl2promela/stop_condition/parser.py

```diff
@@ -80,7 +82,9 @@ def _parse_selector(selector: antlr3.tree.CommonTree) -> Selector:
         elements = elements + nested.elements
     elif selector.children[0].type == lexer.PRIMARY:
         variable_reference = _parse_primary_expression(selector.children[0].children[0])
-        elements.append(variable_reference)
+        elements.append(
```

Comment:
Does it do anything from our perpective (e.g., linter works better?)

"typing.cast(typ, val)[¶](https://docs.python.org/3/library/typing.html#typing.cast)
Cast a value to a type.

This returns the value unchanged. To the type checker this signals that the return value has the designated type, but at runtime we intentionally don’t check anything (we want this to be as fast as possible)."

Reply:
Yes, the `mypy` checker complains about this